filters/openpolicyagent/openpolicyagent.go

```diff
@@ -289,6 +326,37 @@ func (registry *OpenPolicyAgentRegistry) startCleanerDaemon() {
 	}
 }
 
+func (registry *OpenPolicyAgentRegistry) startPluginTriggerDaemon() {
+	interval := registry.pluginTriggerInterval
+
+	// Prevent pods from triggering plugins (f.ex. downloading new bundles) at the same time
+	if registry.maxPluginTriggerJitter > 0 {
+		jitter := time.Duration(rand.Int63n(registry.maxPluginTriggerJitter.Milliseconds())) * time.Millisecond
+		interval += jitter
+	}
+
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+
+	registry.mu.Lock()
+	instances := make([]*OpenPolicyAgentInstance, 0, len(registry.instances))
```

Comment:
This list is dynamic, and the for loop below would run on outdated instances...

Reply:
yes but that avoids having a look that spans http calls
also an instance coming up will be triggered on startup

Reply:
I misunderstood, ofc. moving it into the ticker loop is the right thing to do here.

Reply:
addressed